package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/autorest.modeler",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
Current modeler is 2.3.45, I'm just curious why you do that (IOW, I don't say it's wrong, I'm just curious :))

Reply:
O_o Yeah, we shouldn't bother setting the third digit -- that gets replaced with the git commit count at publish time.